public/util/tagManagerApi.js

```diff
@@ -57,9 +85,10 @@ export default {
   },
 
   createTag: (tag) => {
+    const clean = cleanTag(tag);
```

Comment:
I did consider this but I thought it best to not change anything from under React since this isn't anything *wrong* per se with the React, just how the tagmanager server side processes things.

Would be trivial to move it into an action if you'd prefer.